runtime/gc_base/IndexableObjectAllocationModel.cpp

```diff
@@ -122,9 +144,14 @@ MM_IndexableObjectAllocationModel::initializeIndexableObject(MM_EnvironmentBase
 	GC_ArrayObjectModel *indexableObjectModel = &extensions->indexableObjectModel;
 	J9IndexableObject *spine = (J9IndexableObject*)initializeJavaObject(env, allocatedBytes);
 	_allocateDescription.setSpine(spine);
+	bool isArrayletDataAdjacentToHeader = false;
+	bool isAllIndexableDataContiguousEnabled = indexableObjectModel->isVirtualLargeObjectHeapEnabled();
```

Comment:
isAllIndexableDataContiguousEnabled is confusing name, now that we removed support for double mapping